### ISSUE TITLE
Add additional tools

### ISSUE THREAD

**Issue by Atrate** [csicar/Ning#4]
Another nice feature that Ning could have would be a simple suite of network diagnosis tools, such as ping, traceroute, geoping, WHOIS, DNS Lookup, geoip and WoL.

**Comment by csicar** [comment]
Yes, that's a good idea. At the moment, I don't have the time to implement that, but I would be happy to take contributions
